lemarche/api/urls.py

```diff
@@ -22,6 +27,7 @@
 router.register(r"networks", NetworkViewSet, basename="networks")
 router.register(r"perimeters/kinds", PerimeterKindViewSet, basename="perimeter-kinds")
 router.register(r"perimeters/autocomplete", PerimeterAutocompleteViewSet, basename="perimeters-autocomplete")
+router.register(r"cities/autocomplete", CityAutocompleteViewSet, basename="cities-autocomplete")
```

Comment:
```suggestion
router.register(r"perimeters/cities/autocomplete", PerimeterCityAutocompleteViewSet, basename="perimeters-cities-autocomplete")
```
vu que cities est un type de périmètre

Reply:
ou même mieux réutiliser rajouter simplement un filtre kind dans `PerimeterAutocompleteFilter` pour éviter de créer un nouvel endpoint ?? :innocent:

Reply:
Merci pour la suggestion ! Il y avait d'ailleurs un conflit côté JS 😉